scalameta/semantic/shared/src/main/protobuf/semanticdb.proto

```diff
@@ -31,10 +31,9 @@ message Range {
 
 message Message {
   enum Severity {
-    UNKNOWN = 0;
-    INFO = 1;
-    WARNING = 2;
-    ERROR = 3;
+    INFO = 0;
```

Comment:
Protobuf doesn't allow counting from 1 for enums. I'll put back UNKNOWN and fail it during the deserialization step.